cumulusci/tasks/create_package_version.py

```diff
@@ -41,6 +41,10 @@
     format_subscriber_package_version_where_clause,
 )
 
+PERSISTANT_ORG_ERROR = """
+Target org scratch org definition file missing. Persistent orgs e.g.,Dev Hub can't be used for 2GP package uploads.
+"""
```

Comment:
Minor typo and simplification:
```suggestion
PERSISTENT_ORG_ERROR = """
Target org scratch org definition file missing. Persistent orgs like a Dev Hub can't be used for 2GP package uploads.
"""
```